HotelReservation/HotelReservation/Hotel/HotelData/HiltonHotelData.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using HotelReservation.Mail;
+using HotelReservation.Payment;
+
+namespace HotelReservation.Hotel.HotelData
+{
+    public class HiltonHotelData : HotelData
```

Comment:
To nie jest wg mnie poprawne użycie dziedziczenia. Ta klasa nic tak na prawdę nie rozszerza. Co gorsza taki mechanizm nie pozwala na dynamiczne dodanie hotelu - za każdym razem trzeba tworzyć nową klasę. Duplikujesz też informacje z Hotel.cs i musisz je potem mapować. Takie rozwiązania się faktycznie sprawdzają w większych projektach - tzn oddzielenie modelu z danymi (DTO) od obiektu domenowego - ale tutaj nie masz jeszcze takiej potrzeby.

Reply:
Fixed - usunięte